### PR TITLE
buildSrcVersions + refreshVersions 0.6.3

### PR DESCRIPTION
**Pull request by jmfayard**
See https://github.com/jmfayard/buildSrcVersions/issues/77

### CODE REVIEW

File: buildSrc/src/main/kotlin/Deps.kt [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+ *   `$ ./gradlew buildSrcVersions`
+ */
+object Deps {
+    const val de_fayard_buildsrcversions_gradle_plugin: String =
+            "de.fayard.buildSrcVersions:de.fayard.buildSrcVersions.gradle.plugin:0.6.2"
+
+    const val androidx_test_core_ktx: String = "androidx.test:core-ktx:1.1.0"
+
+    const val kotlinx_coroutines_android: String =
+            "org.jetbrains.kotlinx:kotlinx-coroutines-android:1.2.1"
```
Comment (LouisCAD) [inline buildSrc/src/main/kotlin/Deps.kt]: Why are you adding this file? I thought the versions only mode only used `.properties` files.

File: gradle.properties [hunk 2]
```diff
@@ -23,3 +23,47 @@ android.useAndroidX=true
...
+# Generated by $ ./gradlew refreshVersions
+# You can edit the rest of the file, it will be kept intact
+# See https://github.com/jmfayard/buildSrcVersions/issues/77
+plugin.com.github.ben-manes.versions=0.25.0
+plugin.de.fayard.buildSrcVersions=0.6.2
+version.androidx.test..core-ktx=1.1.0
+#                   # available=1.2.0
+version.androidx.test..runner=1.1.1
+#                 # available=1.2.0
+version.org.jetbrains.kotlinx=1.3.0
```
Comment (LouisCAD) [inline gradle.properties]: Should be kotlinx.coroutines. Non coroutines kotlinx artifacts have different versions. Can you add kotlinx.serialization with its current version to ensure it works correctly?

File: settings.gradle.kts [hunk 3]
```diff
@@ -1,5 +1,15 @@
 @file:Suppress("SpellCheckingInspection")
+pluginManagement {
+    repositories {
+        mavenLocal()
+        gradlePluginPortal()
+    }
 
+    /** TODO: Update to Gradle 5.6.2 then use the boilerplate from here
```
Comment (LouisCAD) [inline settings.gradle.kts]: Is there a reason you're not targeting the `develop` branch which is way ahead? It's already using Gradle 5.6.2

File: buildSrc/src/main/kotlin/Deps.kt (hunk 1 above)
Comment (jmfayard) [reply buildSrc/src/main/kotlin/Deps.kt]: You are totally right. 
I was too much focused on fixing a nasty bug that I forgot what I was supposed to do. I will close this PR and send a new one :)

File: gradle.properties (hunk 2 above)
Comment (jmfayard) [reply gradle.properties]: Good point, I did this in the newer PR

**Pull request closed without merging**